firefox_accounts/views/fxa_flow_aggregates.view.lkml

```diff
@@ -0,0 +1,174 @@
+# note, this is intended to be a temporary solution for counting registrations and logins in a way that can be segmented by entrypoint and UTM parameters
+# we are working on refactoring the FxA tables more generally.
+
+view: fxa_flow_aggregates {
+  derived_table: {
+    interval_trigger: "24 hours"
+    increment_key: "flow_start_date"
+    # go back 2 days to allow recent flows to complete
+    increment_offset: 2
+    sql: WITH
+          flow_ids AS (
+              SELECT
+                  flow_id,
+                  ANY_VALUE(entrypoint) AS entrypoint,
+                  ANY_VALUE(utm_source) AS utm_source,
+                  ANY_VALUE(utm_medium) AS utm_medium,
+                  ANY_VALUE(utm_term) AS utm_term,
+                  ANY_VALUE(utm_campaign) AS utm_campaign,
+                  ANY_VALUE(country) AS country,
+                  ANY_VALUE(ua_browser) AS browser_name,
+                  ANY_VALUE(ua_version) AS browser_version,
+                  ANY_VALUE(os_name) AS os_name,
+                  MIN(DATE(timestamp)) AS flow_start_date,
+              FROM firefox_accounts.fxa_content_auth_oauth_events
+              WHERE DATE(timestamp) >= DATE(2022,9,1)
+              AND event_type = 'fxa_email_first - view'
+              GROUP BY 1
+          ),
+
+      flow_agg AS (
+        SELECT
+          f.*,
```

Comment:
it would be preferable to explicitly list the columns here and in the GROUP BY, otherwise if a field gets added in the base table this query will break and would need to be fixed manually